jetbot/bot.py

```diff
@@ -496,6 +496,25 @@ async def channels(ctx: commands.Context):
                                 channel = await channel_methods[channel_config.get("type", "text")](category)(channel_config["name"])
                                 new.append(channel)
 
+                            # Create or update forum tags if it's a forum channel and has available_tags
```

Comment:
Should this also handle removing existing tags that aren't in the config?

Reply:
oh that's a good idea, sure I can add those